services/get-message.yml

```diff
@@ -1,3 +1,17 @@
+getCurrentTime:
```

Comment:
We should probably delete this get-message.yml file (it's okay for local testing, but it really has no place in the "finished product"